Shared/ActiveDirectoryFunctions/Get-ExchangeProtocolContainer.ps1

```diff
@@ -0,0 +1,22 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\Get-OrganizationContainer.ps1
+
+function Get-ExchangeProtocolContainer {
+    [CmdletBinding()]
+    [OutputType([System.DirectoryServices.DirectoryEntry])]
+    param (
+        [string]$ComputerName = $env:COMPUTERNAME
+    )
+
+    [array]$ComputerName = $ComputerName.Split(".")
+    if ($ComputerName.Count -gt 1) {
+        $ComputerName = $ComputerName[0]
+    }
```

Comment:
This isn't really needed, you can just do the following. 

```suggestion
    $ComputerName = $ComputerName.Split(".")[0]
```

As this will always work. 

![image](https://user-images.githubusercontent.com/22776718/199998015-2796f3ee-433a-436b-b01c-dee380e93fae.png)

Reply:
Fixed